app/src/main/java/hm/orz/chaos114/android/androidsaddlersample/MainActivity.java

```diff
@@ -5,12 +5,17 @@
 import android.view.Menu;
 import android.view.MenuItem;
 
+import android.util.Log;
+
 public class MainActivity extends AppCompatActivity {
 
+    public static final String constant = "test value";
+
     @Override
-    protected void onCreate(Bundle savedInstanceState) {
+    protected void onCreate(Bundle savedInstanceState){
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        constant.substring(1);
```

Comment:
ERROR: [CORRECTNESS][RV_RETURN_VALUE_IGNORED] Return value of String.substring(int) ignored in hm.orz.chaos114.android.androidsaddlersample.MainActivity.onCreate(Bundle)
http://findbugs.sourceforge.net/bugDescriptions.html#RV_RETURN_VALUE_IGNORED